### PR TITLE
Update for Tone Analyzer API v4

### PR DESCRIPTION
**Pull request by richieverma**
https://github.com/watson-developer-cloud/tone-analyzer-nodejs/issues/108

Updated demo has been deployed to https://tone-analyzer-demo-r-v4.mybluemix.net/ for testing purposes.

### CODE REVIEW

**Comment by richieverma** [comment]
Here is a screenshot of the two proposed changes in the following situation:

When there are no dominant tones to be displayed at sentence level, then should the legend block (which depends on tone score) be displayed?

1. Displayed
legendblockdisplayed

2. Not Displayed
legendblocknotdisplayed

**Comment by richieverma** [comment]
Also, wanted to confirm if we can continue using the existing color scheme in the demo? Or do we need to switch to some other color scheme?

**Comment by jzhang300** [comment]
@richieverma up to you, if you think there's a need to update it.

File: manifest.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ declared-services:
 applications:
 - services:
   - my-tone-analyzer-service
-  name: tone-analyzer-demo
+  name: tone-analyzer-demo-r-v4
```
[the remark this answers is not in the document]
Comment (richieverma) [reply manifest.yml]: Updated.

**Comment by richieverma** [comment]
Added test cases and completed cross-browser testing.

**Pull request merged**